File: RESTful Samples/RestfulEngine_JavaClient/src/client/TemplateVariable.java
package client;

/**
 * Created by Bassem on 4/12/2015.
 */
public class TemplateVariable {

    public String name,value;

    public TemplateVariable(String name,String value){
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: RESTful Samples/RESTfulEngine.PHPClient/TemplateVariable.php
<?php
/*
 * Copyright (c) 2015 by Windward Studios, Inc. All rights reserved.
 *
 * This software is the confidential and proprietary information of
 * Windward Studios ("Confidential Information").  You shall not
 * disclose such Confidential Information and shall use it only in
 * accordance with the terms of the license agreement you entered into
 * with Windward Studios, Inc.
 */

/** The template variable.
 */
class TemplateVariable {

	/** Creates a new instance of the variable with name and initial value.
	 */
	public function __construct($name, $value)
	{
		$this->state["name"] = $name;
		$this->state["value"] = $value;
	}

	/** Returns the name of this variable.
	 */
	public function getName()
	{
		return $this->state["name"];
	}

	/** Returns the value of this variable.
	 */
	public function getValue()
	{
		return $this->state["value"];
	}

	/** Assigns a new value to this variable.
	 */
	public function setValue($value)
	{
		$this->state["value"] = $value;
	}
}

?>
